docs/specviz/notebook.rst

```diff
@@ -1,5 +1,20 @@
 ***********************************
-Using SpecViz in a Jupyter Notebook 
+Using Specviz in a Jupyter Notebook 
 ***********************************
 
-Words go here...
+Specviz is developed to fully support analyzing spectra within your existing `Jupyter notebooks <https://jupyter.org/>`_! To use Specviz, install jdaviz in your notebook's python environment and add a new cell wherever you would like to use Specviz.
+::
+
+    # Import specviz
+    from jdaviz import specviz
+    # Instantiate an instance of Specviz
+    specviz = SpecViz()
+    # Display Specviz
+    specviz.app
+
+To extract the data currently loaded into the viewer:
+::
+
+    specviz.app.get_data_from_viewer('spectrum-viewer')
+
+To learn how to add data and use Specviz, including the corresponding API calls, please see our other Specviz Documentation
```

Comment:
"." at the end of this sentence and looks like this file needs a newline at the end as well.

Reply:
Is there an equivalent to these commands to retrieve regions from viewer in cubeviz, that might need documentation?:
region1=cubeviz.app.get_subsets_from_viewer("spectrum-viewer")
or
region1=cubeviz_data.get_selection_definition(format='astropy-regions')

Reply:
There will also presumably be a method to retrieve model-fitting parameters at some point.

Reply:
for `get_subsets_from_viewer` I think it's the exact same code as the above: you use the specviz helper to access the cubeviz helper anyway.  Although, now that you say it I'm realizing we really should make that easier!  But I think that requires addition of a little helper (I'll make an issue about that shortly).


re: `get_selection_definition`, I'm less sure about that. But I think in `cubeviz` selelction is a bit more frought so probably the disucssion belongs there?

Reply:
To clarify the above: I think we do need the thinkgs @PatrickOgle  mentioned, but I think it's best addressed in a follow-on PR rather than this one since it probably requires some code modifications.

Reply:
I agree with @eteq 

Also @javerbukh I see the blank line at the end of the file on my side?
![image](https://user-images.githubusercontent.com/25206008/95492484-23192a80-0969-11eb-9c6d-f80410163246.png)